tests/mappers/test_action_mapper.py

```diff
@@ -14,15 +14,33 @@
 # limitations under the License.
 """Tests action mapper"""
 import unittest
+from unittest import mock
+from typing import Tuple, List, cast
+
+from xml.etree import ElementTree as ET
 
 from o2a.converter.relation import Relation
 from o2a.converter.task import Task
 from o2a.mappers.action_mapper import ActionMapper
+from o2a.o2a_libs.property_utils import PropertySet
+from o2a.utils.config_extractors import TAG_CONFIGURATION, TAG_JOB_XML
+from o2a.utils.xml_utils import find_node_by_tag, find_nodes_by_tag
 
 TEST_MAPPER_NAME = "mapper_name"
 TEST_DAG_NAME = "dag_name"
 
 
+class TestMapper(ActionMapper):
+    def to_tasks_and_relations(self) -> Tuple[List[Task], List[Relation]]:
```

Comment:
Is this the standard way of stubbing methods in Python tests?

Reply:
ActionMapper is an abstract class. We need a specific implementation to be able to test. It is normal to create a test/fixture class in this file.

Reply:
:+1: